vitest.config.ts

```diff
@@ -6,7 +6,6 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     test: {
-      setupFiles: ['src/installSesLockdown.ts'],
```

Comment:
The fake timers don't work with lockdown, so I moved this out of the global setup.